cypress/integration/MetaData/MetaDataTests/index.js

```diff
@@ -0,0 +1,67 @@
+// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+
+//This step is just a workaround until it gets fixed
+Given('user is navigating to {string}', (a) => {
+    Cypress.on('uncaught:exception', (err, runnable) => {
+        // we expect a 3rd party library error with message like 'list not defined' etc.
+        // and don't want to fail the test so we return false
+        if (err.message.includes(`Cannot read property 'getAttribute' of undefined`)) {
+            return false
+        }
+    });
+    cy.visit(a);
+});
+
+And('the page contains og:site_name meta tags with the following properties', (dataTable) => {
+    for (const { property, content } of dataTable.hashes()) {
+        if (content.includes('{CANONICAL_HOST}')) {
```

Comment:
is og:site_name ever going to include cannonical host? this logic is for tags values that are built from path and host, 
I don't believe site_name is one of them

Reply:
Updated per review.